machines/vcItem.ts

```diff
@@ -42,7 +42,7 @@ const model = createModel(
     storeVerifiableCredential: null as VerifiableCredential,
     requestId: '',
     isVerified: false,
-    isPinned: false,
+    isPinned: null as boolean,
     lastVerifiedOn: null,
```

Comment:
as will perform type assertion thus null as boolean is equivalent false. If so, why dont we explicitly say `isPinned: false`
Is my understanding right here?